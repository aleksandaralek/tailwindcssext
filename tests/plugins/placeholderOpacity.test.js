import { quickPluginTest } from '../util/run'

quickPluginTest('placeholderOpacity', {
  safelist: [
    // Arbitrary values
    'placeholder-opacity-[0.12]',
    'placeholder-opacity-[var(--my-value)]',
  ],
}).toMatchSnapshot()
