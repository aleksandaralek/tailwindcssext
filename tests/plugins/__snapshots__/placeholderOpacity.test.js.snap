// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test the 'placeholderOpacity' plugin 1`] = `
"
.placeholder-opacity-0::placeholder {
  --tw-placeholder-opacity: 0;
}

.placeholder-opacity-10::placeholder {
  --tw-placeholder-opacity: .1;
}

.placeholder-opacity-100::placeholder {
  --tw-placeholder-opacity: 1;
}

.placeholder-opacity-15::placeholder {
  --tw-placeholder-opacity: .15;
}

.placeholder-opacity-20::placeholder {
  --tw-placeholder-opacity: .2;
}

.placeholder-opacity-25::placeholder {
  --tw-placeholder-opacity: .25;
}

.placeholder-opacity-30::placeholder {
  --tw-placeholder-opacity: .3;
}

.placeholder-opacity-35::placeholder {
  --tw-placeholder-opacity: .35;
}

.placeholder-opacity-40::placeholder {
  --tw-placeholder-opacity: .4;
}

.placeholder-opacity-45::placeholder {
  --tw-placeholder-opacity: .45;
}

.placeholder-opacity-5::placeholder {
  --tw-placeholder-opacity: .05;
}

.placeholder-opacity-50::placeholder {
  --tw-placeholder-opacity: .5;
}

.placeholder-opacity-55::placeholder {
  --tw-placeholder-opacity: .55;
}

.placeholder-opacity-60::placeholder {
  --tw-placeholder-opacity: .6;
}

.placeholder-opacity-65::placeholder {
  --tw-placeholder-opacity: .65;
}

.placeholder-opacity-70::placeholder {
  --tw-placeholder-opacity: .7;
}

.placeholder-opacity-75::placeholder {
  --tw-placeholder-opacity: .75;
}

.placeholder-opacity-80::placeholder {
  --tw-placeholder-opacity: .8;
}

.placeholder-opacity-85::placeholder {
  --tw-placeholder-opacity: .85;
}

.placeholder-opacity-90::placeholder {
  --tw-placeholder-opacity: .9;
}

.placeholder-opacity-95::placeholder {
  --tw-placeholder-opacity: .95;
}

.placeholder-opacity-\\[0\\.12\\]::placeholder {
  --tw-placeholder-opacity: .12;
}

.placeholder-opacity-\\[var\\(--my-value\\)\\]::placeholder {
  --tw-placeholder-opacity: var(--my-value);
}
"
`;
